src/Type/Php/InArrayFunctionTypeSpecifyingExtension.php

```diff
@@ -110,7 +110,7 @@ public function specifyTypes(FunctionReflection $functionReflection, FuncCall $n
 			$context->true()
 			|| (
 				$context->false()
-				&& count($arrayValueType->getFiniteTypes()) === 1
+				&& count($arrayValueType->getFiniteTypes()) > 0
```

Comment:
1) Tests are failing
2) There are more places in this file that still do `=== 1`, maybe they could be also `> 0`.

Reply:
1. I know, will check tomorrow
2. Looking at the diff that broke it, it feels like a typo only here. The other place was === 1 even before. But I will check it tomorrow too.